server/src/main/java/org/apache/cassandra/sidecar/utils/AuthUtils.java

```diff
@@ -62,4 +66,21 @@ public static List<String> extractIdentities(User user)
         }
         return Collections.unmodifiableList(identities);
     }
+
+    /**
+     * Extracts a list of cassandra roles associated with given user.
+     *
+     * @param user User representation in Vertx
+     * @return {@code List} of cassandra roles associated with user
+     */
+    public static List<String> extractCassandraRoles(User user)
+    {
+        // noinspection unchecked
+        JsonArray roles =  user.attributes().getJsonArray(CASSANDRA_ROLES_ATTRIBUTE_NAME, EMPTY_JSON_ARRAY);
+        if (roles == null)
+        {
+            return List.of();
+        }
+        return roles.getList();
```

Comment:
can `roles` be `null`? can we change this to this instead if that's the case?
```suggestion
        return Objects.requireNonNullElse(user.attributes().getJsonArray(CASSANDRA_ROLES_ATTRIBUTE_NAME), EMPTY_JSON_ARRAY).getList();
```

Reply:
It can be `null` if someone stores null for `CASSANDRA_ROLES_ATTRIBUTE_NAME` key